database/postgres/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
In the PR https://github.com/kubearchive/kubearchive/pull/401/files#diff-0d8d9d8bb96db992801d14fd0d9ef717539dd570c17eafda6ac8c3d6d339df42 I proposed locating these files under: `integrations/database/postgresql/` instead of `database/postgresql`

Reply:
Moved everything to `integrations/database/postgresql`.